### PR TITLE
Null values fix

### PR DESCRIPTION
**Pull request by slehan**
Let me know what you think about the test I made, it does pass all other tests, including the one I made.

### CODE REVIEW

**Comment by slehan** [comment]
Submitted to fix issue #18

File: src/test/scala/com/crealytics/spark/excel/IntegrationSuite.scala [hunk 1]
```diff
@@ -83,4 +84,40 @@ class IntegrationSuite extends FunSuite with PropertyChecks with DataFrameSuiteB
...
+      val expectedWithNull = expected.withColumn("aString", lit(null: String))
+      // Generate the same DataFrame but with empty strings
+      val expectedWithEmptyStr = expected.withColumn("aString", lit("": String))
+      // Set the schema so that aString is nullable
+      expectedWithEmptyStr.schema.fields.update(6, new StructField("aString", DataTypes.StringType, true))
+
+      val tempFile: File = File.createTempFile("spark_excel_null_test_", ".xlsx")
+      val fileName = tempFile.getAbsolutePath
+
+      expectedWithNull.write
```
Comment (nightscape) [inline src/test/scala/com/crealytics/spark/excel/IntegrationSuite.scala]: Could you extract a method like `writeThenRead(df: DataFrame): DataFrame` from line 104-118 and also reuse that in the existing test-case?
Just to keep the duplication down and make the actual intent of the test clearer.

Comment (ManiRz) [inline src/test/scala/com/crealytics/spark/excel/IntegrationSuite.scala]: When we convert a excel file into a dataframe, if the first row contains null in the last column, it is excluding the entire column. Is there any way to fix this? Thanks.

Comment (nightscape) [reply src/test/scala/com/crealytics/spark/excel/IntegrationSuite.scala]: Can you create a new issue for this?

**Pull request merged**